docs/weave/weave-api-spec/02-transaction-signatures.md

```diff
@@ -11,7 +11,7 @@ In this section, we will explain how to sign __transactions__.
 To execute desired actions and state changes on weave based blockchains, every transaction must be validated, authenticated and then authorized to perform the action. In the realm of blockchains this is achieved by using signatures.
 
 Weave uses `ed25519` signing algorithm to achieve this important feature.
-After following the steps in [Weave transactions](transaction.md), created **Tx** must be signed. But first Weave *conditions* role in signing must be explained.
+After following the steps in [Weave transactions](weave-transaction-spec), created **Tx** must be signed. But first Weave *conditions* role in signing must be explained.
```

Comment:
I don't think that this change belongs in this PR.

Reply:
This line caught my eye and I said to myself well PR title has refactoring so this can go in, But youre right this belongs to another PR